lerna.json

```diff
@@ -1,6 +1,7 @@
 {
   "version": "independent",
   "npmClient": "npm",
+  "useNx": "fasle",
```

Comment:
What is this doing? It is a misspelled string so I would assume nothing. Is NX causing issues?

Reply:
Sorry, this is a local debugging code accidentally committed. I have restored the code here.